boards/arm/mimxrt1015_evk/doc/index.rst

```diff
@@ -0,0 +1,210 @@
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+``boards/arm/mimxrt1015_evk/mimxrt1015_evk_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and I/Os
+====================
+
+The MIMXRT1015 SoC has five pairs of pinmux/gpio controllers.
+
++---------------+-----------------+---------------------------+
+| Name          | Function        | Usage                     |
++===============+=================+===========================+
+| GPIO_AD_B0_05 | GPIO            | LED                       |
++---------------+-----------------+---------------------------+
+| GPIO_EMC_09   | GPIO            | SW4                       |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_06 | LPUART1_TX      | UART Console              |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_07 | LPUART1_RX      | UART Console              |
++---------------+-----------------+---------------------------+
+| GPIO_EMC_32   | LPUART4_TX      | UART Console              |
++---------------+-----------------+---------------------------+
+| GPIO_EMC_33   | LPUART4_RX      | UART Console              |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B1_15 | LPI2C1_SDA      | I2C SDA                   |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B1_14 | LPI2C1_CLK      | I2C SCL                   |
++---------------+-----------------+---------------------------+
+
+System Clock
+============
+
+The MIMXRT1015 SoC is configured to use the 24 MHz external oscillator on the
+board with the on-chip PLL to generate a 500 MHz core clock.
+
+Serial Port
+===========
+
+The MIMXRT1015 SoC has four UARTs. ``LPUART1`` is configured for the console,
+and the remaining are not used.
+
+Programming and Debugging
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Configuring a Debug Probe
+=========================
+
+A debug probe is used for both flashing and debugging the board. This board is
+configured by default to use the :ref:`opensda-daplink-onboard-debug-probe`,
+however the :ref:`pyocd-debug-host-tools` do not yet support programming the
```

Comment:
We don't have such a Zephyr document label? Maybe you mean `` :ref:`nxp_opensda_pyocd` ``